spec/system/users_spec.rb

```diff
@@ -92,7 +92,6 @@
         it 'shows users' do
           visit users_path(status: user.user_status_id)
           expect(page).to have_content user.to_s
-          expect(page).to have_content external_user.to_s
```

Comment:
I removed this because previously we were using the same name for each user in the factory but it was switched out to use Faker. The above expectation was a false positive since every user has the same name.

Reply:
I added it back and made it work. Good catch!